environments/routed-networks.yaml

```diff
@@ -21,6 +21,6 @@ parameter_defaults:
   dhcp_relay_image: CentOS-7-x86_64-GenericCloud
 
 resource_registry:
-  OS::OVB::BaremetalNetworks: templates/baremetal-networks-routed.yaml
-  OS::OVB::DHCPRelay: templates/dhcp-relay.yaml
-  OS::OVB::UndercloudNetworks: templates/undercloud-networks-routed.yaml
+  OS::OVB::BaremetalNetworks: ../templates/baremetal-networks-routed.yaml
```

Comment:
Hmm, since this has parameters in it I was actually expecting users to make a copy. Maybe we should split this env file into the resource_registry that can be used directly from environments and the parameters that can be overridden in a user env.

Reply:
Ok, I reverted this in commit bc309fe0fcd39883ebe872585f60ea9bde4e0b6a
Doc's updated to make a copy: commit 4b74d2e1e9240c824323561557d2619a5219a5d9